database/migrations/platform/2023_10_30_000000_update_player_achievements_table.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class() extends Migration {
+    public function up(): void
+    {
+        Schema::table('player_achievements', function (Blueprint $table) {
+            $table->index(
+                ['user_id', 'unlocked_at', 'unlocked_hardcore_at', 'achievement_id'],
+                'player_achievements_user_date_achievement'
```

Comment:
Without a custom index name, an error is thrown for the auto-generated default name being too long.